src/sentry/incidents/models.py

```diff
@@ -164,12 +164,10 @@ class Incident(Model):
     )
     type = models.PositiveSmallIntegerField()
     title = models.TextField()
+    sub_title = models.TextField()
```

Comment:
I think we'd want to allow null here or add a default.

For migrating, I'd recommend allowing this be null, then backfilling, then adding the `NOT NULL` constraint back in if desired. Alternatively, we could add the column with a default, but this has more risks since it takes longer.